samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/SerializationTests.razor

```diff
@@ -0,0 +1,75 @@
+﻿@page "/serialization-tests"
+@using System.Text.Json.Serialization
+@inject IJSRuntime JsRuntime
+
+<h1>SerializationTests</h1>
+<button @onclick="SendJson">Send JSON Serialization</button>
+<button @onclick="SendJsonStream">Send JSON Stream Serialization</button>
+<button @onclick="SendProtobufStream">Send Protobuf Stream Serialization</button>
+@code {
+    protected override void OnInitialized()
+    {
+        _graphics = GenerateGraphics();
+    }
+
+    protected override async Task OnAfterRenderAsync(bool firstRender)
+    {
+        if (firstRender)
+        {
+            _arcGisJsInterop = await JsRuntime.InvokeAsync<IJSObjectReference>("import", 
+                "./_content/dymaptic.GeoBlazor.Core/js/arcGisJsInterop.js");
+            await _arcGisJsInterop.InvokeVoidAsync("loadProtobuf");
+        }
+    }
+
+    private async Task SendJson()
+    {
+        long unixTimeMilliseconds = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
+        await _arcGisJsInterop!.InvokeVoidAsync("getFromJson", _graphics, unixTimeMilliseconds);
+    }
+    
+    private async Task SendJsonStream()
+    {
+        JsonSerializerOptions options = new()
+        {
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
+            PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+            PropertyNameCaseInsensitive = true
+        };
+        long unixTimeMilliseconds = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
+        MemoryStream ms = new MemoryStream(JsonSerializer.SerializeToUtf8Bytes(_graphics, options));
+        DotNetStreamReference streamRef = new(ms);
+        await _arcGisJsInterop!.InvokeVoidAsync("getFromJsonStream", streamRef, unixTimeMilliseconds);
+    }
+
+    private async Task SendProtobufStream()
```

Comment:
does this need to exist if its commented out?

Reply:
It's a placeholder for the next step, where I need to replace all feature `queries` with protobuf serialization. I will add comments to it.

Reply:
Nope, sorry, you were right, this is old garbage. Deleting.